doc/components.md

```diff
@@ -134,14 +141,16 @@ You can also change the current playing animation using the `updateAnimation` me
 
 For a working example, check this [source file](/doc/examples/flare/lib/main_component.dart).
 
-## Composed component
+## Composability of components
```

Comment:
I think this should be a `#` (1) level topic after the list of components. it is not one of the classes anymore to be on this list. also could be a `###` topic within BaseComponent

Reply:
You can't have several level 1 topics if you want to follow the markdown standard, so I'll do your other suggestion